src/lib/API/api.js

```diff
@@ -29,6 +30,23 @@ export type UserRecord = NameRecord &
     username?: string,
   }
 
+export const getErrorMessage = apiError => {
+  let { message } = apiError
+
+  // if the json or string http body was thrown from axios (error
+  // interceptor in api.js doest that in almost cases) then we're wrapping
+  // it onto Error object to keep correct stack trace for Sentry reporting
+  if (!isError(apiError)) {
+    message = apiError.error || apiError
+  }
+
+  if (message) {
+    apiError.message = message
+  }
```

Comment:
```suggestion
```

no need to mutale because a) it's wrong because apiError in most cases isn't an Error object b) we have to create errors anyway in the place where we're logging